### PR TITLE
Book crud and permissions

### PR DESCRIPTION
**Pull request by AVyha**
I tried to do this views with DRY principle and add JWT token with custom permissions, give please feedback about this variant.

### CODE REVIEW

File: book/views.py [hunk 1]
```diff
@@ -1,30 +1,13 @@
...
-class BookRetrieveUpdateDestroy(generics.RetrieveUpdateDestroyAPIView):
-    serializer_class = BookSerializer
-    permission_classes = [permissions.IsAuthenticated]
-
-    def get_queryset(self):
-        user = self.request.user
-
-        return Book.objects.filter(user=user)
+    queryset = Book.objects.all()
+    authentication_classes = (JWTAuthentication,)
```
Comment (mltdll) [inline book/views.py]: `JWTAuthentication` is already on by default. :)

**Review by mltdll — CHANGES_REQUESTED** [review]
GJ! I thought that it's probably better to implement all the necessary things first, and then think about the optimization and refactoring, but it's great that you did it anyway. But there's one minor thing to fix. :)

File: book/models.py [hunk 2]
```diff
@@ -17,7 +17,8 @@ class Book(models.Model):
     inventory = models.PositiveIntegerField()
     daily_fee = models.DecimalField(
         max_digits=5,
-        decimal_places=2
+        decimal_places=2,
+        null=True
```
Comment (mltdll) [inline book/models.py]: I also can't really see what this `null=True` is for. Can you clarify this?

Comment (AVyha) [reply book/models.py]: hmmm you right 👍

Comment (mltdll) [reply book/models.py]: Should this line be deleted then?

Comment (AVyha) [reply book/models.py]: yes

Comment (AVyha) [reply book/models.py]: i'm trying 😊

**Review by mltdll — APPROVED** [review]

**Review by olia-trofymchuk — APPROVED** [review]

**Pull request merged**